portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase

```diff
@@ -440,6 +440,64 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-11509 | Test Summary: Check Document Rating show expected value"
```

Comment:
Hey @calvinctvo I think in the place of  `Feature ID` you should put the bug ticket, like this: `@description = "Bug: LRAC-123`

Reply:
I will resolve this since Vini already pointed out :rofl: